CheckYourEligibility-Admin/Views/Check/ApplicationsRegistered.cshtml

```diff
@@ -32,25 +37,23 @@
                 @foreach (var item in @Model.Children)
                 {
                     <tr class="govuk-table__row">
-                        <td class="govuk-table__cell">@item.ParentName</td>
                         <td class="govuk-table__cell">@item.ChildName</td>
                         <td class="govuk-table__cell">@item.Reference</td>
                     </tr>
                 }
-
             </tbody>
         </table>
     </div>
 </div>
+
 <div class="govuk-width-container">
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-full">
             <h2 class="govuk-heading-m">What to do next</h2>
             <p> @TempData["Status"]</p>
```

Comment:
I think having an if clause here based on the application status will be better than saving the full string in memory. Do you need to see an example of how it could work?

Reply:
Yeah okay, I can refactor that. I had just returned the code that was originally there from the file history. I removed it previously without realising it was used here. That said, I think I know what you're suggesting and if the text isn't used anywhere else we may as well just store the outcome value in TempData and store the text in the view.